src/routes/mylist/+page.svelte

```
@@ -0,0 +1,697 @@
+                                        <form
+                                            method="POST"
+                                            action="?/removeWatchlist"
+                                            use:enhance={() => {
+                                                return async () => {
+                                                    await invalidateAll();
+                                                };
+                                            }}
+                                        >
+                                            <input type="hidden" name="anime_id" value={anime.id} />
+                                            <button type="submit" class="remove-btn" aria-label="リストから削除" title="リストから削除">
+                                                <svg width="13" height="13" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"><polyline points="3 6 5 6 21 6"/><path d="M19 6l-1 14a2 2 0 0 1-2 2H8a2 2 0 0 1-2-2L5 6"/><path d="M10 11v6"/><path d="M14 11v6"/><path d="M9 6V4h6v2"/></svg>
```

Biome (2.4.13)

[error] Alternative text title element cannot be empty

(lint/a11y/noSvgWithoutTitle)